cmd/osquery-perf/agent.go

```diff
@@ -851,6 +858,9 @@ func main() {
 	templateNames := []string{
 		"mac10.14.6.tmpl",
 
+		// Uncomment this to add windows hosts
+		// "windows_11.tmpl",
```

Comment:
Not for now, but would be nice to make `osquery-perf` configurable so that we can have e.g. say 33% hosts Windows, 33% Linux and 33% macOS.